packages/jsapi-nodejs/src/NodeHttp2gRPCTransport.ts

```diff
@@ -177,7 +177,6 @@ export class NodeHttp2gRPCTransport implements GrpcTransport {
     logger.debug('cancel');
     assertNotNull(this.request, 'request is required');
     this.request.close();
-    this.request = null;
```

Comment:
@vbabich I ended up reverting this as Colin pointed out this breaks the check in the `start` method to avoid calling start multiple times. We could make the state logic smarter, but I think it will introduce complexity that can't easily be tested. Probably best to leave simple for now, unless we run into any particular problems.